formatter/sarif_test.go

```diff
@@ -26,24 +26,26 @@ func Test_sarifPrint(t *testing.T) {
 			Issues: tflint.Issues{},
 			Stdout: fmt.Sprintf(`{
   "version": "2.1.0",
-  "$schema": "https://json.schemastore.org/sarif-2.1.0-rtm.5.json",
+  "$schema": "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/main/sarif-2.1/schema/sarif-schema-2.1.0.json",
   "runs": [
     {
       "tool": {
         "driver": {
+          "informationUri": "https://github.com/terraform-linters/tflint",
           "name": "tflint",
-          "version": "%s",
-          "informationUri": "https://github.com/terraform-linters/tflint"
+          "rules": [],
```

Comment:
Are empty rules valid? 🤔

Reply:
From https://github.com/oasis-tcs/sarif-spec/blob/123e95847b13fbdd4cbe2120fa5e33355d4a042b/Schemata/sarif-schema-2.1.0.json#L3045 yes an empty rule block is valid. I had to add this as the new report included an empty rules block on generation, instead of omitting it in the rc version.

Reply:
Thank you for the explanation. Indeed, this seems to be valid even with the [SARIF online validator](https://sarifweb.azurewebsites.net/Validation).